src/GridSquare.svelte

```diff
@@ -5,12 +5,31 @@
   export let isActive: boolean;
   export let isPartOfCurrentClue: boolean;
   export let square: Square;
+  export let updateCell: (newValue: string) => void;
```

Comment:
If typescript was nice you could have a type of chars not strings, but it doesn't seem so ergonomic:

https://stackoverflow.com/a/71623113 is probably the best way.